IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CommunicationsApplianceTypeTransportEquipment/DocPropertySet.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_CommunicationsApplianceTypeTransportEquipment" UniqueId="861ced4b-d3ff-d4f2-e3ea-93b0d3b64717" ApplicableType="IfcCommunicationsAppliance/TRANSPORTEQUIPMENT" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="AveragePowerOfSingleChannel_3rB_Tjhz6S8Z3ZXo0VSc6" />
+		<DocProperty xsi:nil="true" href="IsUpgradable_2ZU8jJnSr22RzVeas8FkPD" />
+		<DocProperty xsi:nil="true" href="ChromaticDispersionTolerance_1GC7m4ctzEMR6sRVs2Rji" />
+		<DocProperty xsi:nil="true" href="ElectricalCrossCapacity_0quq4gG9n9tv6gtXkQaQF" />
+		<DocProperty xsi:nil="true" href="MaxPowerOfSingleChannel_25tcvQvv7RRphaUrbQGbb" />
+		<DocProperty xsi:nil="true" href="MinOpticalSignalToNoiseRatio_0rhrcZcL2wgrIvyycBvf" />
+		<DocProperty xsi:nil="true" href="MinPowerOfSingleChannel_3F01Zd0gfCOQlFpSU3C2Fh" />
+		<DocProperty xsi:nil="true" href="NumberOfOccupliedSlots_1M6YNdqvn6C9dvFOhu18Vj" />
```

Comment:
Instance Property?

Reply:
Same consideration as above.

Reply:
Since this is happening quite often, could we somehow look into grouping these properties or maybe get the product based properties from the manufacturer?